static/app/utils/formatters.tsx

```diff
@@ -176,12 +176,28 @@ export function getDuration(
   return `${label} ${tn('millisecond', 'milliseconds', result)}`;
 }
 
+const SUFFIX_ABBR = {
+  years: t('yr'),
```

Comment:
These might be difficult to translate without context but I don't think we have any way to provide it with our locale utils 😞